admin_tests/registration_providers/test_views.py

```diff
@@ -166,6 +167,34 @@ def test_share_source(self, mock_share, view, provider, req):
         assert provider.share_source == 'test source'
         assert provider.access_token == 'test access token'
 
+    @mock.patch.object(settings, 'SPAM_FLAGGED_MAKE_NODE_PRIVATE', 'testenv')
+    def test_share_source_prefix(self, mock_share, view, provider, req):
```

Comment:
You should check the requests in this test too.  `responses.calls[-1].request` for the most recent request.